DependencyInjection/MonologExtension.php

```diff
@@ -336,6 +336,14 @@ private function buildHandler(ContainerBuilder $container, $name, array $handler
                 ));
                 $container->setDefinition($handlerId.'.not_found_strategy', $activationDef);
                 $activation = new Reference($handlerId.'.not_found_strategy');
+            } elseif (!empty($handler['excluded_http_codes'])) {
+                $activationDef = new Definition('Symfony\Bridge\Monolog\Handler\FingersCrossed\HttpCodeActivationStrategy', array(
```

Comment:
we should probably throw an exception is the class does not exist (perhaps even in the config builder)?